rpctest/blockgen.go

```diff
@@ -125,8 +125,8 @@ func createBlock(prevBlock *btcutil.Block, inclusionTxs []*btcutil.Tx,
 	// it's the genesis block in which case use the current time.
 	var ts time.Time
 	switch {
-	case blockTime != nil:
-		ts = *blockTime
+	case !blockTime.IsZero():
+		ts = blockTime
 	case blockHeight == 0:
```

Comment:
Hmm, yeah you're correct. Seems this was left over from the original gist
this was pulled from. I've removed this case entirely.

On Fri, Aug 19, 2016 at 4:25 PM Dave Collins <notifications@github.com>
wrote:

> In rpctest/blockgen.go
> <https://github.com/btcsuite/btcd/pull/371#discussion_r75553686>:
>
> > @@ -125,8 +125,8 @@ func createBlock(prevBlock *btcutil.Block, inclusionTxs []*btcutil.Tx,
> >  	// it's the genesis block in which case use the current time.
> >  	var ts time.Time
> >  	switch {
> > -	case blockTime != nil:
> > -		ts = *blockTime
> > +	case !blockTime.IsZero():
> > +		ts = blockTime
> >  	case blockHeight == 0:
>
> Shouldn't this be impossible? You always have to build off at least the
> genesis block which is at height and given blockHeight is prevBlock.Height()
> + 1, that means it must always be at least 1.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/btcsuite/btcd/pull/371/files/2925fbd6bf40359007399e1bc5295f12f66cf9aa..2a3c3a04f6351c7e2a7e1d719fe77eb5a201ef6f#r75553686>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AA87LsAXg1OZL0a0P6sZoN4A207MQqukks5qhh88gaJpZM4D7zxE>
> .
>